libindy/indy-wallet/src/storage/default/transaction.rs

```diff
@@ -14,6 +14,7 @@ impl<'conn> Transaction<'conn> {
             TransactionBehavior::Deferred => "BEGIN DEFERRED",
             TransactionBehavior::Immediate => "BEGIN IMMEDIATE",
             TransactionBehavior::Exclusive => "BEGIN EXCLUSIVE",
+            _ => ""
```

Comment:
There's no other variants, this should be removed, looks dangerous in case the struct would change in the future, and this match branch would then cover unintended scenarios